lib/Network/OAuth2/Server/API.hs

```diff
@@ -57,26 +84,41 @@ import           Network.OAuth2.Server.UI
 logName :: String
 logName = "Anchor.Tokens.Server.API"
 
+
+-- TODO: Move this into some servant common package
+
+-- | http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9.2
+--
+-- The purpose of the no-store directive is to prevent the inadvertent release
+-- or retention of sensitive information (for example, on backup tapes).
 data NoStore = NoStore
 instance ToByteString NoStore where
     builder _ = "no-store"
 
+-- | Same as Cache-Control: no-cache, we use Pragma for compatibilty.
```

Comment:
https://tools.ietf.org/html/rfc6749#section-5.1

The authorization server MUST include the HTTP "Cache-Control"
   response header field [RFC2616] with a value of "no-store" in any
   response containing tokens, credentials, or other sensitive
   information, as well as the "Pragma" response header field [RFC2616]
   with a value of "no-cache".